packages/storybook/src/avatar/css-avatar.stories.tsx

```diff
@@ -0,0 +1,167 @@
+    >
+      {imageSrc ? (
+        <img src={imageSrc} alt={altText} className="kernteam-avatar__image" />
+      ) : children ? (
+        children
+      ) : privateProfile ? (
+        <IconUser className="kernteam-avatar__icon" />
+      ) : (
+        <IconBuilding className="kernteam-avatar__icon" />
+      )}
+    </button>
+  );
+};
+
+const meta = {
+  title: 'CSS Component/Avatar',
+  id: 'css-avatar',
+  component: Avatar,
+  argTypes: {
+    children: {
+      name: 'initials',
+      description:
+        'Choose initials. We recommend a maximum of 2 characters. When it is left empty the default for this story is an icon.',
+      type: {
+        name: 'string',
+      },
+      table: {
+        category: 'Children',
+      },
+    },
+    borderRadius: {
+      name: '--kernteam-avatar-border-radius',
+      description: 'Set the design token for avatar-border-radius',
+      type: {
+        name: 'string',
+      },
+      table: {
+        category: 'Design Tokens',
+      },
+    },
+    backgroundColor: {
+      name: '--kernteam-avatar-background-color',
+      description: 'Set the design token for avatar-background-color',
+      type: {
+        name: 'string',
+      },
+      table: {
+        category: 'Design Tokens',
+      },
+    },
+    color: {
+      name: '--kernteam-avatar-color',
+      description: 'Set the design token for avatar-color',
+      type: {
+        name: 'string',
+      },
+      table: {
+        category: 'Design Tokens',
+      },
+    },
+    imageSrc: {
+      name: 'avatar image',
+      description: 'Set the image for avatar--image. If this value is set, any children will not be shown.',
+      control: {
+        type: 'file',
+        accept: '.png,.svg,.jpg,.gif',
+      },
+      table: {
+        category: 'Children',
+      },
+    },
+    altText: {
+      name: 'alt-text',
+      description: 'Set the alt text for the avatar. Currently used als alt for image',
+      type: {
+        name: 'string',
+      },
+      table: {
+        category: 'Children',
+      },
+    },
+    privateProfile: {
+      name: 'private or company',
+      description: 'Set the icon-variant for the avatar.',
+      value: {
+        control: { type: 'radio' },
+        options: ['Private profile', 'Company profile'],
+      },
+      type: {
+        name: 'boolean',
+      },
+      table: {
+        category: 'Children',
+      },
+    },
+  },
+  args: {
+    borderRadius: '50%',
+    backgroundColor: 'red',
+    color: 'white',
+    altText: '',
+    privateProfile: true,
+  },
+  tags: ['autodocs'],
+  parameters: {
+    docs: {
+      description: {
+        component: readme,
+      },
+    },
+  },
+} satisfies Meta<typeof Avatar>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {};
+
+export const Image: Story = {
+  args: {
+    imageSrc: 'https://entertainment.inquirer.net/files/2012/08/beyonce-knowles.jpg',
+    altText: 'Profielfoto',
```

Comment:
Zouden de properties korter kunnen en meer in lijn met de DOM APIs?
`<Avatar src="some.png" alt="Pretty Woman walking down the street"/>`